How to choose security protocols is out of scope, but cite RFC 8922 (#1442)
ietf-tapswg/api-drafts: draft-ietf-taps-interface.md

Of course any communication over a network reveals usage characteristics, because all
packets, as well as their timing and size, are part of the network-visible wire image {{?RFC8546}}. However,
the selection of a protocol and its configuration also impacts which information is visible, potentially in

> clear text, and which other entities can access it. How Transport Services systems ought to choose protocols depending on the needed level or type of security is out of scope of the present document as well as {{?I-D.ietf-taps-impl}}, as these documents are limited to transport protocols. The choice of a security protocol can be informed by the survey provided in {{?RFC8922}}.

Author: ```suggestion
clear text, and which other entities can access it. How Transport Services systems ought to choose protocols depending on the needed level or type of security is out of scope of the present document as well as {{?I-D.ietf-taps-impl}}, because the current documents are limited to transport protocols. The choice of a security protocol can be informed by the survey provided in {{?RFC8922}}.
```

> 

Reviewer: /as these documents are limited/because the current documents are limited/

> clear text, and which other entities can access it. How Transport Services systems ought to choose protocols depending on the needed level or type of security is out of scope of the present document as well as {{?I-D.ietf-taps-impl}}, because the current documents are limited to transport protocols. The choice of a security protocol can be informed by the survey provided in {{?RFC8922}}.

Reviewer 2: ```suggestion
clear text, and which other entities can access it. How Transport Services systems ought to choose protocols depending on the security properties required is out of scope of this specification, as it is limited to transport protocols. The choice of a security protocol can be informed by the survey provided in {{?RFC8922}}.
```
